.github/workflows/github.yml

```diff
@@ -129,6 +135,7 @@ jobs:
             artifact_platform: arm64-v8a
 
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ !startsWith(matrix.platform, 'msvc-') }}
```

Comment:
Leftovers from testing?
Or this is needed in some cases? If yes then comment would be useful